### PR TITLE
Fixed typo / changed latest stable to 6.0

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

**Comment by MrGeneration** [comment]
Seems like you got confused here too?
The 5.1 part is an example only. I wanted to have an example for people to understand the path but didn't want to keep updating this exact  phrase on every release...

File: install/update.rst [hunk 1]
```diff
@@ -19,11 +19,11 @@ Updating Zammad
       for your and your customers safety.
 
       In case you couldn't update for a longer time, please ensure to at least
-      update from **mayor to mayor** version. Big version jumps *may* work but
+      update from **major to major** version. Big version jumps *may* work but
       usually go terribly wrong. As example, expecting the current stable
       version of Zammad being 5.1 and your instance being on Zammad 2.4, your
       path would look like so:
-      ``2.4`` → ``3.0`` → ``4.0`` → ``5.0`` → ``latest stable (5.1)``
+      ``2.4`` → ``3.0`` → ``4.0`` → ``5.0`` → ``latest stable (6.0)``
```
Comment (mgruner) [inline install/update.rst:26]: This is not correct now, the text says 'being 5.1'. Maybe you can just keep this section as before, or update both places.

File: install/update.rst [hunk 2]
```diff
@@ -19,11 +19,11 @@ Updating Zammad
       for your and your customers safety.
```
Comment (mgruner) [inline install/update.rst:19]: `for customers'`

**Review by mgruner — APPROVED** [review]
Nice fixes, just minor suggestions.

**Pull request merged**